admin_manual/configuration_files/external_storage/openstack.rst

```diff
@@ -2,21 +2,27 @@
 OpenStack Object Storage
 ========================
 
-OpenStack Object Storage is used to connect to an OpenStack Swift server.
-Two authentication mechanisms are available: one is the generic OpenStack
-mechanism, and the other is used exclusively for Rackspace, a provider of object
-storage that uses the OpenStack Swift protocol.
-
-The bucket will be created if it does not exist.
```

Comment:
This statement is still true, and important for some users where a bucket is not explicitly given by the Swift provider